res/skins/Deere/style.qss

```diff
@@ -1595,6 +1595,10 @@ WPushButton[value="2"]:hover {
     border: 0px solid #0080BE;
 }
 
+#HotcueButton {
+    qproperty-shouldHighlightBackgroundOnHover: true;
```

Comment:
This doesn't seem to be working. I see no color change when hovering hotcue buttons in Deere that have a hotcue set.

Reply:
This is weird, it works for me :/
What Qt version are you linking with?

Reply:
5.12.5

Reply:
This probably depends on the QStyle in use... Did you test with `./mixxx -style fusion`?